src/index.tsx

```diff
@@ -49,10 +49,7 @@ export * from './components/DataLabel';
 export { default as Textarea } from './components/Textarea';
 export { default as EmojiPicker } from './components/EmojiPicker';
 export * from './components/Title';
-export { default as Panel } from './components/Panel';
-export { default as PanelHeading } from './components/PanelHeader';
```

Comment:
I just realized the default export name is `PanelHeading`, we should use PanelHeading everywhere then

Reply:
Missed that one, will fix in a bit.